hw03_mukalamusij.py

```diff
@@ -0,0 +1,104 @@
+# TODO Do not edit this file directly! Instead, create a new file called
+# TODO a03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Joyce Mukalamusi
+# Username: mukalamusij
+#
+# Assignment: HW03: Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose:
+# Google Doc Link: https://docs.google.com/document/d/16yXRh2zAV79Lqzp0jjNsU5dQf5AVqUKSO7DxrRJjoeo/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements:
+#
+#
+#################################################################################
+
+
+import turtle as h
+
+# Set up the screen
+h.bgcolor("skyblue")
```

Comment:
This is interesting that it works. What you've done is called `bgcolor()` on the turtle library (which you renamed to `h`), which surprises me that it works. That's because `bgcolor()` is a method of the `Screen()` class in the turtle library (not a function in the **turtle.py** file)

What you should have done is:

```
import turtle
wn = turtle.Screen()
wn.bgcolor("skyblue")
```

The `import` statement belongs here at the beginning of the code. The two `wn` lines belong in `main()`, as the first two lines inside of the `main()` function.

Reply:
Ok I just changed it but I also called in my turtle with the h as well. So, rename my turtle to "turtle.Turtle()"?